aspnetcore/tutorials/first-web-api.md

```diff
@@ -67,6 +67,9 @@ A NuGet package must be added to support the database used in this tutorial.
 * From the **Tools** menu, select **NuGet Package Manager > Manage NuGet Packages for Solution**.
 * Select the **Browse** tab.
 * Enter **Microsoft.EntityFrameworkCore.InMemory** in the search box, and then select `Microsoft.EntityFrameworkCore.InMemory`.
+* Select the **Project** checkbox in the right pane and then select **Install**.
+* Enter **Swashbuckle.AspNetCore.SwaggerUI** in the search box, and then select `Swashbuckle.AspNetCore.SwaggerUI`.
```

Comment:
We're no longer using swagger. The .NET 9 controller API template generates the following code:

```
var builder = WebApplication.CreateBuilder(args);

// Add services to the container.

builder.Services.AddControllers();
// Learn more about configuring OpenAPI at https://aka.ms/aspnet/openapi
builder.Services.AddOpenApi();

var app = builder.Build();

// Configure the HTTP request pipeline.
if (app.Environment.IsDevelopment())
{
    app.MapOpenApi();
}

app.UseHttpsRedirection();

app.UseAuthorization();

app.MapControllers();

app.Run();
```

Reply:
I resolved offline with Rick.  The statement "We're no longer using swagger." was not correct as a broad statement.    The Swashbuckle package tools are no longer used to generate the OpenAPI doc.  SwaggerUI is still used and recommended for generating UI for interactivity.  See Safia's https://learn.microsoft.com/en-us/aspnet/core/fundamentals/openapi/using-openapi-documents?view=aspnetcore-9.0#use-swagger-ui-for-local-ad-hoc-testing